vmm/src/vm.rs

```diff
@@ -319,6 +319,9 @@ pub enum Error {
 
     #[error("Error injecting NMI")]
     ErrorNmi,
+    #[cfg(feature = "mshv")]
+    #[error("Error unfreezing the VM: {0}")]
+    UnfreezeVm(#[source] hypervisor::HypervisorVmError),
```

Comment:
Since you renamed freeze/unfreeze to pause/resume maybe we don't need this anymore?